### PR TITLE
List of Employee bought rewards.

### PR DESCRIPTION
**Pull request by konrad04-1989**
Hi,

below is a recording of the new feature
[Loom](https://www.loom.com/share/03bf9f98d1544a498f0e01b57a7462a7)

### CODE REVIEW

File: app/controllers/admin/employees_controller.rb [hunk 1]
```diff
@@ -4,12 +4,13 @@ module Admin
...
     # GET /admin/employees
     def index
-      @employees = Employee.all.includes([:orders])
+      @employees = Employee.all.includes([:orders]).sort
     end
 
     # GET /admin/employees/1
     def show
       find_employee
+      @orders = Order.all.where(employee: @employee)
```
Comment (ciejj) [inline app/controllers/admin/employees_controller.rb]: why not - you can also get rid of that, and simply call `@employee.orders` in the view
```suggestion
      @orders = @employee.orders
```

File: spec/system/reward_buying_system_spec.rb [hunk 2]
```diff
@@ -56,6 +56,19 @@
     expect(page).to have_text(reward.price)
     expect(reward.price) == 1
   end
+
+  it 'enables Admin to see list of rewards bought by the employee' do
+    create(:kudo, receiver_of_kudo: employee)
+    sign_in(employee)
+    visit(rewards_path)
+    find('tr', text: reward.title).click_on('Buy')
+    sign_in(admin_user)
```
Comment (ciejj) [inline spec/system/reward_buying_system_spec.rb]: I would get rid of that part - you have the reward purchase already covered in the other test, and simply use a factory for reward, and an employee

**Review by ciejj — APPROVED** [review]
nice one! ☝️ left couple of suggestions for improvement

File: app/controllers/admin/employees_controller.rb (hunk 1 above)
Comment (konrad04-1989) [reply app/controllers/admin/employees_controller.rb]: Thanks! Changed ;)

File: spec/system/reward_buying_system_spec.rb (hunk 2 above)
Comment (konrad04-1989) [reply spec/system/reward_buying_system_spec.rb]: Changed. I created an order factory and a new spec file to test the new feature. Thanks for the advice!

**Pull request merged**